### PR TITLE
Ex4-TDD

### PR DESCRIPTION
**Pull request by matanan**
@ItamarShDev 
@robi-y

### CODE REVIEW

File: tests/find-a-person.js [hunk 1]
```diff
@@ -6,6 +6,6 @@ describe('Find a person', function() {
   it('Given a person name, return all posts (of a map) containing her name (in any of a post fields)', function() {
     var map = new Map(["I met Or A. at Chabad house Bangkok", "We found Or A. R.I.P at Langtang valley", "Random post"]);
     var posts = map.find_a_person("Or A.")
-    expect(posts).to.be.eql(["I met Or A. at Chabad house Bangkok", "We found Or A. R.I.P at Langtang valley"]);
+    expect(posts).to.be.eql(["I met Or A. at Chabad house Bangkokk", "We found Or A. R.I.P at Langtang valley"]);
```
Comment (ItamarShDev) [inline tests/find-a-person.js:9]: why failing the test when you need to fail the method itself?

**Comment by ItamarShDev** [comment]
> Given a name, check if the map includes a location information for it (a place or geo. location), 
> 
> Check if there are map inconsistencies, e.g., the same name with different locations.

those two where missing

**Comment by matanan** [comment]
@ItamarShDev

**Comment by ItamarShDev** [comment]
:+1:

**Pull request closed without merging**